doc/make/msvc.md

```diff
@@ -0,0 +1,40 @@
+Compiling Lean with Visual Studio
+---------------------------------
+
+WARNING: Compiling Lean with Visual Studio doesn't currently work.
+There's an ongoing effort to port Lean to Visual Studio.
+The instructions below are for VS 2017.
+
+
+## Installing dependencies
+
+First, install `vcpkg` from https://github.com/Microsoft/vcpkg if you haven't
+done so already.
+Then, open a console in the directory you cloned `vcpkg` to, and type:
+`vcpkg install mpir` for the 32-bit library or
+`vcpkg install mpir:x64-windows` for the x64 one.
+
+In Visual Studio, use the "open folder" feature and open the Lean directory.
+Go to the `CMake->Change CMake Settings` menu. File `CMakeSettings.json` opens.
+In each of the targets, add the following snippet (i.e., after every
+`ctestCommandArgs`):
+
+```json
+      "variables": [
+        {
+          "name": "CMAKE_TOOLCHAIN_FILE",
+          "value": "C:\\path\\to\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake"
+        }
+      ]
+```
+
+
+## Enable Intellisense
+
```

Comment:
Which version of Visual Studio are you using?
I tried with VS 2017 and Win 10. Everything works up to this point.
`Ctrl+Q` does nothing on my version. `F7` also didn't work.
I had to select `CMake > Build all` to build Lean.

Reply:
I also have VS 2017 (15.5)
Ctrl+Q just places the focus of the cursor in the search box at the top right of the windows; it doesn't do anything on its own.
I found a bug in VS that makes this Ctrl+Q feature not always find these cmake json files. I've reported it already.

Reply:
> Ctrl+Q just places the focus of the cursor in the search box at the top right of the windows; it doesn't do anything on its own.

Got it. I found the search box :)

BTW, it is still building, it is super slow :)